metrics/metrics.go

```diff
@@ -0,0 +1,18 @@
+package metrics
+
+// Start spins up the metrics server
+func Start() {
+	// TODO add to config
+	httpPort := 8000
+	httpEnabled := true
+
+	if httpEnabled {
```

Comment:
Should we support https?

Reply:
Well i was thinking that it shouldn't be mandatory to run the server? Alternatively, you might only want the prom server (in the future) and not the http?